IdSSLOpenSSL.pas

```diff
@@ -243,8 +243,8 @@ interface
   TIdSSLAction = (sslRead, sslWrite);
 
 const
-  DEF_SSLVERSION = sslvTLSv1;
-  DEF_SSLVERSIONS = [sslvTLSv1];
+  DEF_SSLVERSION = sslvTLSv1_3;
+  DEF_SSLVERSIONS = [sslvTLSv1_3];

```

Comment:
I would suggest enabling at least TLS 1.2 by default as well. Maybe TLS 1.1, too? (https://github.com/IndySockets/Indy/issues/181). But, having multiple versions in `DEF_SSLVERSIONS` would require `DEF_SSLVERSION` to be `sslvSSLv23` instead, at least until the `SSLOptions.Method` property is eventually eliminated.